### PR TITLE
[Doc] Update spill_mem_limit desc

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Why I'm doing:

What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function
- [ ] This is a backport pr

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto-backported to the target branch
  - [x] 3.2
  - [x] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/administration/resource_group.md [hunk 1]
```diff
@@ -38,9 +38,11 @@ displayed_sidebar: "Chinese"
...
 - `concurrency_limit`：资源组中并发查询数的上限，用以防止并发查询提交过多而导致的过载。只有大于 0 时才生效，默认值为 0。
 - `max_cpu_cores`：资源组在单个 BE 节点中使用的 CPU 核数上限。仅在设置为大于 `0` 后生效。取值范围：[0, `avg_be_cpu_cores`]，其中 `avg_be_cpu_cores` 表示所有 BE 的 CPU 核数的平均值。默认值为 0。
-- `spill_mem_limit_threshold`: 资源组在当前 BE 节点触发落盘的内存占用阈值（百分比）。取值范围：[0,1]，默认值为 1。该参数自 v3.1.7 版本引入。
-  - 如果查询开启落盘功能且命中资源组（包括默认资源组 `default_wg`），那么如果该查询使用的内存超过 `query_mem_limit * mem_limit * spill_mem_limit_threshold`，则会触发中间结果落盘以减少内存占用。其中 `query_mem_limit` 为当前 BE 节点上查询的内存限制，由系统变量 `query_mem_limit` 控制。
-  - 如果查询开启落盘功能，但不受资源组功能管理，系统将根据其实际内存占用决定是否触发中间结果落盘。
+- `spill_mem_limit_threshold`: 当前资源组触发落盘的内存占用阈值（百分比）。取值范围：(0,1)，默认值为 1，即不生效。该参数自 v3.1.7 版本引入。
+  - 如果开启自动落盘功能（即系统变量 `spill_mode` 设置为 `auto`），但未开启资源组功能，系统将在查询的内存占用超过 `query_mem_limit` 的 80% 时触发中间结果落盘。其中 `query_mem_limit` 为单个查询可使用的内存上限，由系统变量 `query_mem_limit` 控制，默认值为 0，代表不设限制。
+  - 如果开启自动落盘功能且查询命中资源组（包括所有系统内建资源组）后，该查询使用的内存超过以下任意一项，都将触发中间结果落盘：
+    - 当前资源组内所有查询使用的内存超过 `当前 BE 节点内存上限 * mem_limit * spill_mem_limit_threshold` 时
+    - 当前查询占用 `query_mem_limit` 的 80% 时
```
Comment (evelynzhaojie) [inline docs/zh/administration/resource_group.md:45]: 当前查询占用的内存 达到/超过 `query_mem_limit` 的 80% 时

File: docs/en/administration/resource_group.md [hunk 2]
```diff
@@ -70,12 +70,13 @@ You can specify CPU and memory resource quotas for a resource group on a BE by u
...
-  - If spilling is enabled for a query but the query is not managed by the resource group feature, the system will decide whether to trigger spilling based on query's actual memory usage.
+  The memory usage threshold (percentage) at which a resource group triggers the spilling of intermediate results. The valid range is (0, 1). The default value is 1, indicating the threshold does not take effect. This parameter was introduced in v3.1.7.
+  - If automatic spilling is enabled (that is, the system variable `spill_mode` is set to `auto`) but the resource group feature is disabled, the system will trigger spilling when the memory usage of a query exceeds 80% of `query_mem_limit`. Here, `query_mem_limit` is the maximum memory that a single query can use, controlled by the system variable `query_mem_limit`, with a default value of 0, indicating no limit.
+  - If automatic spilling is enabled, and the query hits a resource group (including all system built-in resource groups), spilling will be triggered if the query meets any of the following:
```
Comment (evelynzhaojie) [inline docs/en/administration/resource_group.md:77]: any of the following conditions

File: docs/en/administration/resource_group.md [hunk 3]
```diff
@@ -70,12 +70,13 @@ You can specify CPU and memory resource quotas for a resource group on a BE by u
...
+    - when the memory used by all queries in the current resource group exceeds `current BE node memory limit * mem_limit * spill_mem_limit_threshold`
```
Comment (evelynzhaojie) [inline docs/en/administration/resource_group.md:78]: delete "when", use a complete sentence "The memory used by all queries in the current resource group exceeds xxx."

File: docs/en/administration/resource_group.md [hunk 4]
```diff
@@ -70,12 +70,13 @@ You can specify CPU and memory resource quotas for a resource group on a BE by u
...
+    - when the current query consumes more than 80% of `query_mem_limit`
```
Comment (evelynzhaojie) [inline docs/en/administration/resource_group.md:79]: The current query xxx

**Review by evelynzhaojie — APPROVED** [review]

**Review by amber-create — APPROVED** [review]

**Pull request merged**